clusterctl/examples/vsphere/README.md

```diff
@@ -0,0 +1,22 @@
+# Vsphere Example Files
+## Contents
+*.yaml files - concrete example files that can be used as is.
+*.yaml.template files - template example files that need values filled in before use.
+
+## Generation
+For convenience, a generation script which populates templates where possible.
+
+1. Run the generation script.
```

Comment:
can you mention which files are generated by the generation script

Reply:
Done